app/_locales/es/messages.json

```diff
@@ -163,32 +281,94 @@
   "connect": {
     "message": "Conectar"
   },
+  "connectAccountOrCreate": {
+    "message": "Conectar una cuenta o crear una nueva"
+  },
   "connectHardwareWallet": {
     "message": "Conectar Monedero Físico"
   },
+  "connectManually": {
+    "message": "Conectar manualmente al sitio actual"
+  },
+  "connectTo": {
+    "message": "Conectar a $1",
+    "description": "$1 is the name/origin of a web3 site/application that the user can connect to metamask"
+  },
+  "connectToAll": {
+    "message": "Conectar a todos sus $1",
+    "description": "$1 will be replaced by the translation of connectToAllAccounts"
+  },
+  "connectToAllAccounts": {
+    "message": "cuentas",
+    "description": "will replace $1 in connectToAll, completing the sentence 'connect to all of your accounts', will be text that shows list of accounts on hover"
+  },
+  "connectToMultiple": {
+    "message": "Conectar a $1",
+    "description": "$1 will be replaced by the translation of connectToMultipleNumberOfAccounts"
+  },
+  "connectToMultipleNumberOfAccounts": {
+    "message": "$1 cuentas",
+    "description": "$1 is the number of accounts to which the web3 site/application is asking to connect; this will substitute $1 in connectToMultiple"
+  },
+  "connectWithMetaMask": {
+    "message": "Conectar Con MetaMask"
```

Comment:
"Conectar con MetaMask"

Reply:
Yes, however, the [english version](https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json#L314) is "Connect **W**ith Metamask".

Reply:
seems strange but if that is that is how we are using with the brand, then ok.

Reply:
"Connect With Metamask" means the dapp can read your ethereum address